node/src/effect/requests.rs

```diff
@@ -435,6 +456,15 @@ impl<I> Display for ApiRequest<I> {
                 "query {}, base_key: {}, path: {:?}",
                 state_root_hash, base_key, path
             ),
+            ApiRequest::QueryEraValidators {
+                global_state_hash,
+                era_id,
+                ..
+            } => write!(
+                formatter,
+                "auction {}, era_id: {}",
```

Comment:
```suggestion
                "global_state_hash {}, era_id {}",
```

Reply:
this is following the pattern established by the preexisting query and balance variants.

```
ApiRequest::QueryGlobalState {
                state_root_hash,
                base_key,
                path,
                ..
            } => write!(
                formatter,
                "query {}, base_key: {}, path: {:?}",
                state_root_hash, base_key, path
            ),
            ApiRequest::QueryEraValidators {
                state_root_hash,
                era_id,
                ..
            } => write!(formatter, "auction {}, era_id: {}", state_root_hash, era_id),
            ApiRequest::GetBalance {
                state_root_hash,
                purse_uref,
                ..
            } => write!(
                formatter,
                "balance {}, purse_uref: {}",
                state_root_hash, purse_uref
            ),
```